docs/Documentation/Applications/chemicalKinetics.md

```diff
@@ -87,12 +91,12 @@ $ conda deactivate
 
 [Installation: C++ version](https://cantera.org/install/conda-install.html#sec-conda-development-interface)
 ```
-$ module load conda
-$ conda create --name ct-dev --channel cantera libcantera-devel
-$ conda activate ct-dev
+$ module load anaconda3/2024.06.1
+$ conda create --prefix ./ct-dev --channel cantera libcantera-devel
```

Comment:
Maybe include navigating to a projects directory, or include that in the path for the prefix. This is assuming user is already in a /projects directory. (Same goes for examples below)